liubov.tarasenko/task2.rb

```diff
@@ -0,0 +1,20 @@
+def yes
+  hash = {
+    'Yes' => 23,
+    'b' => 'travel',
+    'yesterday' => 34,
+    5 => '234',
+    yesss: :fg,
+    try: 30,
+    key: 'some value', 
+    ' yesterday1 '=> 34,
+    ' yesteryear '=> 2014
+  }
+  count = 0
+  hash.each{|k, v| 
+    count +=1 if k.to_s.downcase.index("yes") == 0
```

Comment:
@lubo4kaTarasenko It would be better to avoid having an extra variable `count`

Method `Enumerable#count` allows passing a block. `hash.keys.count { |key| key.to_s.downcase.index("yes") == 0 }`

Reply:
Ok